explorer/src/assets/locales/en/en.json

```diff
@@ -52,7 +52,7 @@
   "home": {
     "title": "Your Reputation Under Your Control",
     "description": "Explore the ecosystem to find out how you can unlock your data from across the web and share it with the world on your own terms",
-    "exploreEcosystem": "or explore the ecosystem",
+    "exploreEcosystem": "or <bold>explore the ecosystem . . .</bold>",
```

Comment:
What's up with the spacing in the ellipsis?
Suggestion: use ellipsis character "…".